js/components/tooltip.js

```diff
@@ -0,0 +1,114 @@
+var React = require('react');
+var deepmerge = require('deepmerge');
+
+function getContainerStyle(bottom, visible, responsivenessMultiplier, alignment) {
+    var verticalAlignment = 80 * responsivenessMultiplier;
```

Comment:
80 is the height offset between the tooltip and scrubber bar, I agree a constant would make more sense